core/tests/helper.py

```diff
@@ -0,0 +1,14 @@
+# !/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""Helper methods for testing purpose"""
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First line should end with a period, question mark, or exclamation point (not 'e')](https://app.codacy.com/manual/lucas625/Template_Django/pullRequest?prid=5220949)